### PR TITLE
Add yarn in strategies

### PR DESCRIPTION
**Pull request by yhirano55**

### CODE REVIEW

**Comment by yhirano55** [comment]
@sanemat Please review, and tell me so if there is a problem in this PR.

File: lib/tachikoma/application.rb [hunk 1]
```diff
@@ -221,6 +221,23 @@ def cocoapods
       end
     end
 
+    def yarn
+      Dir.chdir("#{Tachikoma.repos_path}/#{@build_for}") do
+        sh(*['git', 'config', 'user.name', @commiter_name])
+        sh(*['git', 'config', 'user.email', @commiter_email])
+        sh(*['git', 'checkout', '-b', "tachikoma/update-#{@readable_time}", @base_remote_branch])
+        sh(*%w(yarn install))
+        sh(*%w(yarn outdated))
```
Comment (sanemat) [inline lib/tachikoma/application.rb]: I'm not familiar with yarn, is this necessary command?

Comment (yhirano55) [inline lib/tachikoma/application.rb]: Thanks, review.
Yes. We have to execute `yarn outdated` before `yarn upgrade`.

see at (ci-yarn-upgrade: keep npm dependencies tool like tachikoma) :
https://github.com/taichi/ci-yarn-upgrade/blob/master/src/upgrade-request.js#L109
^ same process: install -> outdated -> upgrade

**Comment by yhirano55** [comment]
I have resolved conflicts. please review it, again.

**Comment by sanemat** [comment]
Thank you for your contribution!

**Pull request merged**